internal/services/apimanagement/schemaz/api_management.go

```diff
@@ -91,11 +91,6 @@ func SchemaApiManagementOperationRepresentation() *pluginsdk.Schema {
 
 				"form_parameter": SchemaApiManagementOperationParameterContract(),
 
-				"sample": {
```

Comment:
we can't remove this from schema, can we just mark it as deprecated, add the new examples property, and that it'll be removed in 3.0?

Reply:
I think the core issue is that Microsoft have removed it from the SDK, so if I leave it in the schema the build fails with:
```
../services/apimanagement/schemaz/api_management.go:132:4: unknown field 'Sample' in struct literal of type apimanagement.RepresentationContract
../services/apimanagement/schemaz/api_management.go:178:7: v.Sample undefined (type apimanagement.RepresentationContract has no field or method Sample)
../services/apimanagement/schemaz/api_management.go:179:25: v.Sample undefined (type apimanagement.RepresentationContract has no field or method Sample)
```

I can mark the field as deprecated, but it doesn't resolve the compile issues. How has this been handled in the past? Or do we need to wait for v3.0 to be able to upgrade the API Version because Microsoft have introduced a breaking change?

Reply:
Ok - with further testing today, I've identified that whatever is provided as `sample` in the old API is provided within the `examples` array (named default). So my plan is to (attempt to) remove `sample` from the schema but retain it within the terraform and make it a computed attribute (marked as deprecated) and we can then remove it in v3.0.